frontend/components/FilterContent.vue

```diff
@@ -0,0 +1,240 @@
+        <CitySelector class="headfield" />
+        <PaidSelector class="headfield paidSelector" />
+      </div>
+      <hr />
+      <p class="tagTitleText noselect">
+        # Wybierz najważniejszą technologię z tagów
+      </p>
+      <div class="tagsContainer noselect">
+        <div class="tag"><span>React</span></div>
+        <div class="tag"><span>JS</span></div>
+        <div class="tag"><span>Android</span></div>
+        <div class="tag"><span>PHP</span></div>
+        <div class="tag"><span>HTML/CSS</span></div>
+        <div class="tag"><span>Data Analytics</span></div>
+        <div class="tag"><span>C++</span></div>
+        <div class="tag"><span>Python</span></div>
+        <div class="tag"><span>Game Dev</span></div>
+        <div class="tag"><span>Ruby</span></div>
+        <div class="tag"><span>Java</span></div>
+        <div class="tag"><span>.Net</span></div>
+        <div class="tag"><span>Scala</span></div>
+        <div class="tag"><span>Mobile</span></div>
+        <div class="tag"><span>Testing</span></div>
+        <div class="tag"><span>DevOps</span></div>
+        <div class="tag"><span>Security</span></div>
+        <div class="tag"><span>Scala</span></div>
+      </div>
+      <button v-on:click="dropdown" class="approveFilters">
+        Zatwierdź filtry
+      </button>
+    </div>
+    <div v-on:click="dropdown" class="drpDwnBtn noselect">
+      <p class="drpDwmBtnText">Filtrowanie</p>
+    </div>
+  </div>
+</template>
+<script>
+import PaidSelector from "./PaidSelector.vue";
+import CitySelector from "./CityFieldSelector.vue";
+import CourseSelector from "./CourseFieldSelector";
+
+export default {
+  components: {
+    PaidSelector: PaidSelector,
+    CitySelector: CitySelector,
+    CourseSelector: CourseSelector,
+  },
+  props: {
+    reset: Boolean,
+  },
+  data() {
+    return {
+      isActive: false,
+    };
+  },
+
+  methods: {
+    dropdown() {
+      document
+        .querySelector(".drpDwnBtnSect")
+        .classList.toggle("drpDwnBtnSectActive");
+    },
+  },
+  mounted: function () {
+    const tags = document.querySelectorAll(".tag");
+    const arr = [];
+
+    for (let i = 0; i < tags.length; i++) {
+      tags[i].addEventListener("click", function () {
+        if (arr.includes(tags[i].firstElementChild.innerText)) {
+          const index = arr.indexOf(tags[i].firstElementChild.innerText);
+          if (index > -1) {
+            arr.splice(index, 1);
+          }
+        } else {
+          arr.push(tags[i].firstElementChild.innerText);
+        }
+        tags[i].classList.toggle("highlight");
+      });
+    }
+    this.eventBus.on("reset", (reset) => {
+      if (reset) {
+        let high = document.querySelectorAll(".highlight");
+        for (var i = 0; i < high.length; i++) {
+          high[i].classList.toggle("highlight");
+        }
+        arr.length = 0;
+      }
+    });
+  },
+};
+</script>
+
+<style lang="scss" scoped>
```

Comment:
Ask for tailwind, as above.

Reply:
We already created an Issue mentioning transition into tailwind. It will be fixed later on